particle-accelerator/.gitattributes

```diff
@@ -0,0 +1,12 @@
+*       text    eol=lf
```

Comment:
This might be worth putting in the root of the repo. I think it would apply everywhere and the eol thing seems like it might be really useful.

Reply:
Okay, this was part of the boilerplate as well.